### PR TITLE
Switch from ghcr to ecr

### PR DESCRIPTION
**Pull request by anuraaga**
*Issue #, if available:*

*Description of changes:*
Was still using ghcr here, but let's finally switch to ecr. The URL doesn't have to look great for these testing images so it's just a default random alias name

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/release-build.yml [hunk 1]
```diff
@@ -55,22 +60,3 @@ jobs:
...
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
-
-      - name: Login to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
-      - name: Push image to Amazon ECR
```
Comment (willarmiros) [inline .github/workflows/release-build.yml:70]: Using the `jib` command does this push instead now right?

**Review by willarmiros — APPROVED** [review]
LGTM!

File: .github/workflows/main-build.yml [hunk 2]
```diff
@@ -14,14 +14,16 @@ jobs:
...
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
       - name: Login to ECR
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com
-          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          registry: public.ecr.ews
```
Comment (lupengamzn) [inline .github/workflows/main-build.yml:26]: Shouldn't this be `public.ecr.aws`?

Comment (anuraaga) [reply .github/workflows/main-build.yml:26]: It is, isn't it?

Comment (lupengamzn) [reply .github/workflows/main-build.yml:26]: I mean it's `ews` currently

Comment (anuraaga) [reply .github/workflows/main-build.yml:26]: Doh thought I copy-pasted this from somewhere but apparently not, thanks for seeing it!

File: .github/workflows/release-build.yml (hunk 1 above)
Comment (anuraaga) [reply .github/workflows/release-build.yml:70]: Yeah

**Pull request merged**